opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -395,163 +387,216 @@ def test_aggregated_resources_different_schema_urls(self):
-        os.environ[
-            resources.OTEL_RESOURCE_ATTRIBUTES
-        ] = "k=v,k2=v2,invalid,,foo=bar=baz,"
+        detector = OTELResourceDetector()
+        environ[OTEL_RESOURCE_ATTRIBUTES] = "k=v,k2=v2,invalid,,foo=bar=baz,"
         self.assertEqual(
             detector.detect(),
-            resources.Resource({"k": "v", "k2": "v2", "foo": "bar=baz"}),
+            Resource({"k": "v", "k2": "v2", "foo": "bar=baz"}),
         )
 
     def test_multiple_with_url_decode(self):
-        detector = resources.OTELResourceDetector()
-        os.environ[
-            resources.OTEL_RESOURCE_ATTRIBUTES
+        detector = OTELResourceDetector()
+        environ[
+            OTEL_RESOURCE_ATTRIBUTES
         ] = "key=value%20test%0A, key2=value+%202"
         self.assertEqual(
             detector.detect(),
-            resources.Resource({"key": "value test\n", "key2": "value+ 2"}),
+            Resource({"key": "value test\n", "key2": "value+ 2"}),
         )
         self.assertEqual(
             detector.detect(),
-            resources.Resource(
+            Resource(
                 {
                     "key": parse.unquote("value%20test%0A"),
                     "key2": parse.unquote("value+%202"),
                 }
             ),
         )
 
-    @mock.patch.dict(
-        os.environ,
-        {resources.OTEL_SERVICE_NAME: "test-srv-name"},
+    @patch.dict(
+        environ,
+        {OTEL_SERVICE_NAME: "test-srv-name"},
     )
     def test_service_name_env(self):
-        detector = resources.OTELResourceDetector()
+        detector = OTELResourceDetector()
         self.assertEqual(
             detector.detect(),
-            resources.Resource({"service.name": "test-srv-name"}),
+            Resource({"service.name": "test-srv-name"}),
         )
 
-    @mock.patch.dict(
-        os.environ,
+    @patch.dict(
+        environ,
         {
-            resources.OTEL_SERVICE_NAME: "from-service-name",
-            resources.OTEL_RESOURCE_ATTRIBUTES: "service.name=from-resource-attrs",
+            OTEL_SERVICE_NAME: "from-service-name",
+            OTEL_RESOURCE_ATTRIBUTES: "service.name=from-resource-attrs",
         },
     )
     def test_service_name_env_precedence(self):
-        detector = resources.OTELResourceDetector()
+        detector = OTELResourceDetector()
         self.assertEqual(
             detector.detect(),
-            resources.Resource({"service.name": "from-service-name"}),
+            Resource({"service.name": "from-service-name"}),
         )
 
     def test_process_detector(self):
-        initial_resource = resources.Resource({"foo": "bar"})
-        aggregated_resource = resources.get_aggregated_resources(
-            [resources.ProcessResourceDetector()], initial_resource
+        initial_resource = Resource({"foo": "bar"})
+        aggregated_resource = get_aggregated_resources(
+            [ProcessResourceDetector()], initial_resource
```

Comment:
I don't mind this, but it sometimes makes it hard to follow what is the actual change for the original PR vs import path change. Do you want us to follow the `from x import y` vs `import x` and `x.y` Please open an issue, and let's stick to some convention.

Reply:
Ok, from now on I'll refrain from adding these changes in unrelated PRs. :+1: